impl/src/main/java/org/ehcache/impl/internal/store/offheap/LazyOffHeapValueHolder.java

```diff
@@ -124,4 +134,69 @@ private void writeObject(java.io.ObjectOutputStream out) {
     throw new UnsupportedOperationException("This subclass of AbstractValueHolder is NOT serializable");
   }
 
+  private static final boolean IN_TESTING;
```

Comment:
Unless there is absolutely not alternative, I'm very much against testing code in non-testing repos. We should try to move all this out, even if that means modifying the offheap lib to be even more test-friendly.